koku/providers/provider_errors.py

```diff
@@ -91,6 +91,10 @@ class ProviderErrors:
         " and subscription ID or scope with export name. "
         "Edit your Azure source to include these details."
     )
+    AZURE_EXPIRED_CLIENT_SECRET_KEYS_MESSAGE = (
+        "The provided client secret keys for this source are expired."
+        "In Azure, refresh your client secret and try again."
```

Comment:
```suggestion
        "The provided client secret keys for this source are expired. "
        "In Azure, refresh your client secret and try again."
```
This extra space is necessary, else this would appear as
```
The provided client secret keys for this source are expired.In Azure, refresh your client secret and try again.
```

Reply:
gotcha 👍🏿